### PR TITLE
add test case contribution instruction

### PR DESCRIPTION
**Pull request by sunyuan3**
Signed-off-by: Yuan Sun <yile.sy@alibaba-inc.com>

**1.Describe what this PR did**
Add test case contribution instruction in CONTRIBUTING.md.

**2.Does this pull request fix one issue?** 
No

**3.Describe how you did it**
None. Just for updating document. 

**4.Describe how to verify it**
None

**5.Special notes for reviews**
None

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -138,6 +138,13 @@ No matter commit message, or commit content, we do take more emphasis on code re
 
 PR is the only way to make change to Pouch project files. To help reviewers better get your purpose, PR description could not be too detailed. We encourage contributors to follow the [PR template](./.github/PULL_REQUEST_TEMPLATE.md) to finish the pull request.
 
+### Test case contribution
+
+Any test case would be welcomed. Currently, pouch function test cases is a high priority task.
```
Comment (Letty5411) [inline CONTRIBUTING.md]: is -> are

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -138,6 +138,13 @@ No matter commit message, or commit content, we do take more emphasis on code re
...
+
+* For unit test, you need to create a test file ended with `_test.go` in the same directory as dev package.
+* For integration test, you need to add test scrips in `pouch/test/` directory. The test makes use of [package check](https://github.com/go-check/check), a rich testing extension for Go's testing package. Test scripts are named by pouch commands. For example, all pouch help api tests are included in pouch_api_help_test.go and all pouch help tests of commandline tests are included in pouch_cli_help_test.go. For more details, refer to [gocheck document](https://godoc.org/gopkg.in/check.v1).
+
```
Comment (Letty5411) [inline CONTRIBUTING.md]: How about:
"all pouch help tests of commandline tests" -> "all pouch help command line tests"
"refer to" -> please refer to

File: CONTRIBUTING.md (hunk 1 above)
Comment (sunyuan3) [reply CONTRIBUTING.md]: Nice catch.

**Comment by sunyuan3** [comment]
This pr is created in private pouch and it hang.  The underlying bug of github causes this problem. I would close this pr and open another one to workaround.

**Pull request closed without merging**